ci/requirements_all.txt

```diff
@@ -3,6 +3,7 @@ pytest-cov
 pytest-xdist
 flake8
 sphinx
```

Comment:
We also have `requirements_dev.txt`, which I think is more appropriate. I read `requirements_all.txt` as all the optional deps for using pandas (xlrd, xlwt, tables, etc.). Maybe `sphinx` and the test libs should go in `requirements_dev.txt`?